_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -137,18 +146,59 @@ These options map directly to options available in the S3 Select [CSVInput](http
+:--- | :--- | :--- | :---
+`start_time` | No  | String | The start of the last modified time to scan objects from all the buckets. This should follow [ISO LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) format, for example, `023-01-23T10:00:00`, or it can be configured to the keyword which represents current `LocalDateTime`, such as `now`. To define a time range, use `start_time` with  either `end_time` or `range`. 
+`end_time` | No | String  | The end of the last modified time to scan objects from all the buckets. This should follow [ISO LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) format, for example, `023-01-23T10:00:00`, or it can be configured to the keyword which represents current `LocalDateTime`, such as `now`. To define a time range, use `end_time` with  either `start_time` or `range`. 
+`range` | No  | String  | The last modified time range to scan objects from all buckets. If configured with `start_time`, defines a last modified time range with `start_time` + `range`. If configured with `end_time`, defines a last modified time range with `end_time` - `range`. Supports ISO_8601 notation strings, such as `PT20.345S` or `PT15M`, and notation strings for seconds (`60s`) and milliseconds (`1600ms`). 
+`buckets` | Yes  | List | A list of [buckets](#bucket) to scan.
+`scheduling` | No  | List | The configuration for scheduling periodic scans on all buckets. `start_time`, `end_time` and `range` can not be used if scheduling is configured. 
+
+### bucket
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+`bucket` | Yes | String list | Provides options for each bucket.
+
+You can configure the following options inside the bucket setting.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+`name` | Yes | String   | The string representing the S3 bucket name to scan.
+`filter` | No | [Filter](#filter) | Provides the filter configuration.
+`start_time` | No  | String | The start of the last modified time to scan objects from all the buckets. This should follow [ISO LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) format, for example, `023-01-23T10:00:00`, or it can be configured to the keyword which represents current `LocalDateTime`, such as `now`. To define a time range, use `start_time` with  either `end_time` or `range`. 
+`end_time` | No | String  | The end of the last modified time to scan objects from all the buckets. This should follow [ISO LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) format, for example, `023-01-23T10:00:00`, or it can be configured to the keyword which represents current `LocalDateTime`, such as `now`. To define a time range, use `end_time` with  either `start_time` or `range`. 
```

Comment:
```suggestion
`end_time` | No | String  | The end of the last modified time to scan objects from all the buckets. This should follow [ISO LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) format, for example, `023-01-23T10:00:00`, or it can be configured to the keyword that represents the current `LocalDateTime`, such as `now`. To define a time range, use `end_time` with either `start_time` or `range`. 
```